### PR TITLE
[PORT] SuperLube! from TG

### PR DESCRIPTION
**Pull request by ro5490**
## About The Pull Request
Ports TG's SuperLube
https://github.com/tgstation/tgstation/pull/44595
https://github.com/tgstation/tgstation/pull/45134

Added a water flower that sprays superlube instead.
Added superlube as a reagent.

## Why It's Good For The Game

HONK

## Changelog
:cl:nemvar
add: something something lube... HONK
add: Added a new loot item to the lavaland clown ruin.
del: Removed the slime core from said ruin.
tweak: The lavaland clown ruin has some new pranks ready.
/:cl:

### CODE REVIEW

**Comment by utilbird** [comment]
Honk, add the original author's name(s) to the right of the :cl:

**Comment by ro5490** [comment]
> Honk, add the original author's name(s) to the right of the 🆑

Done, I'll be adding the clown ruin changes to this as well
at some point in the next few hours

**Comment by ro5490** [comment]
thats everything

File: code/game/turfs/open/_open.dm [hunk 1]
```diff
@@ -194,7 +194,7 @@
 			if(!(lube&GALOSHES_DONT_HELP)) //can't slip while buckled unless it's lube.
 				return 0
 		else
-			if(!(C.mobility_flags & MOBILITY_STAND) || !(C.status_flags & CANKNOCKDOWN)) // can't slip unbuckled mob if they're lying or can't fall.
+			if(!(lube&SLIP_WHEN_CRAWLING) && (!(C.mobility_flags & MOBILITY_STAND) || !(C.status_flags & CANKNOCKDOWN))) // can't slip unbuckled mob if they're lying or can't fall.
```
Comment (utilbird) [inline code/game/turfs/open/_open.dm]:
```suggestion
			if(!(lube & SLIP_WHEN_CRAWLING) && (!(C.mobility_flags & MOBILITY_STAND) || !(C.status_flags & CANKNOCKDOWN))) // can't slip unbuckled mob if they're lying or can't fall.
```

**Review by utilbird — DISMISSED** [review]

**Comment by DatBoiTim** [comment]
How make super loob? HONK!

**Comment by ro5490** [comment]
Didn't mean to do that lmao

**Review by utilbird — APPROVED** [review]

**Comment by Rukofamicom** [comment]
> How make super loob? HONK!

Hopefully you don't. Regular lube is already super strong.

**Comment by ro5490** [comment]
You cant make super-lube
Unless you get it from the new flower in the ruin, its admin only.

**Comment by hotimep** [comment]
what does super lube do

**Pull request merged**